packages/foreman/foreman/gen-gem-buildreqs

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+require 'bundler'
```

Comment:
This needs `rubygem(bundler)`, but I don't think it's part of our `BuildRequires` today?

```
BuildRequires: rubygem(bigdecimal)
BuildRequires: /usr/bin/ruby
BuildRequires: rubygems
BuildRequires: rubygem(rake) >= 0.8.3
BuildRequires: rubygem(bundler_ext)
```

(rubygems probably pulls it in, or bundler_ext, but I prefer being explicit)

Reply:
I think it's a good point that we become explicit about build requires. I also considered expanding the script with generating a correct Ruby requirement.

Also, I've been wanting to get rid of `bundler_ext`. Its point was to remove the bundler version requirements, but these days we convert them literally. With this it's even more.

Reply:
Added as an explicit build requires.